sdk/identity/identity-persistence/package.json

```diff
@@ -0,0 +1,102 @@
+    "build:samples": "echo skipped",
+    "build:test": "tsc -p . && rollup -c rollup.config.js 2>&1",
+    "build": "npm run extract-api && tsc -p . && rollup -c 2>&1",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\" \"samples/**/*.{js,json}\"",
+    "clean": "rimraf dist dist-* typings *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\" \"samples/**/*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "mocha -r esm -r ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace --exclude \"test/**/browser/**/*.spec.ts\" \"test/**/*.spec.ts\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/identity",
+    "dist-esm/identity-persistence",
+    "types/identity-persistence.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "azure",
+    "cloud",
+    "active directory",
+    "authentication",
+    "credential",
+    "certificate",
+    "managed service identity",
+    "client secret",
+    "access token"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "engine": {
+    "node": ">=8.0.0"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity-persistence/README.md",
+  "sideEffects": false,
+  "dependencies": {
+    "@azure/core-auth": "^1.3.0",
+    "@azure/identity": "^2.0.0-beta.4",
```

Comment:
Yes. It's probably not required for VSCode anymore, but it is required for cache-persistence because it's where we get the options type for the cache from.